### PR TITLE
Localization: Update translated strings

### PR DESCRIPTION
**Pull request by rooftopcellist**
This PR pulls in newly translated strings.  They were pushed on October 15th.  I expect these will need to be backported to 4.4.  

Signed-off-by: Christian M. Adams <chadams@redhat.com>

### CODE REVIEW

File: locale/fr.po [hunk 1]
```diff
@@ -1849,55 +2477,167 @@ msgstr "Voir les conteneurs privés"
...
+msgid "{collectionName}'s versions."
+msgstr "versions de {collectionName}."
+
+#: src/components/cards/collection-card.tsx:44
+#: src/components/cards/namespace-card.tsx:36
+#: src/components/collection-list/collection-list-item.tsx:57
+#: src/components/headers/collection-header.tsx:317
+#: src/components/headers/partner-header.tsx:46
+msgid "{company} logo"
+msgstr "logo {entreprise}"
```
Comment (himdel) [inline locale/fr.po:2631]: `{company}` -> `{enterprise}` .. sounds like we need to explainto translators to never translate these?

(same for `{name}` -> `{nom}` right below)

**Review by himdel — APPROVED** [review]
LGTM, except for those couple of renames in the french version.

@rooftopcellist do you want to fix these in here, or should we merge as is?

**Comment by himdel** [comment]
Written a tool to catch these (adding in TODO PR), but here's the full list:

```
Difference between msgid="<0>Note:</0> Installing collections with ansible-galaxy is only supported in ansible 2.9+" and msgstr="<0>Remarque : L'installation de collections avec ansible-galaxy n'est prise en charge que dans ansible 2.9+":
  Missing from msgstr: </0>
  at ./lint.pl line 30, fr.po line 38.
Difference between msgid="<0>Note:</0> Use this URL to configure ansible-galaxy to upload collections to this namespace. More information on ansible-galaxy configurations can be found <1>here</1>." and msgstr="<0>Remarque : Utilisez cette URL pour configurer ansible-galaxy afin de télécharger des collections dans cet espace de noms. Vous trouverez plus d'informations sur les configurations d'ansible-galaxy ici.":
  Missing from msgstr: </0> <1> </1>
  at ./lint.pl line 30, fr.po line 42.
Difference between msgid="<0>WARNING</0> copy this token now. This is the only time you will ever see it." and msgstr="<0>ATTENTION, copiez ce jeton maintenant. C'est la seule fois que vous le verrez.":
  Missing from msgstr: </0>
  at ./lint.pl line 30, fr.po line 46.
Difference between msgid="<0>WARNING</0> loading a new token will delete your old token." and msgstr="<0>ATTENTION : le chargement d'un nouveau jeton supprimera votre ancien jeton.":
  Missing from msgstr: </0>
  at ./lint.pl line 30, fr.po line 50.
Difference between msgid="<0>{0}</0> will be permanently deleted." and msgstr="<0>{0} sera définitivement supprimé.":
  Missing from msgstr: </0>
  at ./lint.pl line 30, fr.po line 54.
Difference between msgid="<0>{name}</0> will be permanently deleted." and msgstr="<0>{nom} sera définitivement supprimé.":
  Missing from msgstr: {name} </0>
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 58.
Difference between msgid="<0>{username}</0> will be removed from <1>{groupname}</1>." and msgstr="<0>{username} sera retiré de {groupname}.":
  Missing from msgstr: </0> <1> </1>
  at ./lint.pl line 30, fr.po line 62.
Difference between msgid="Deleting <0>{name}</0> and its data will be lost." and msgstr="Suppression de <0>{digest}</0>, et ses données seront perdues.":
  Missing from msgstr: {name}
  Unexpected in msgstr: {digest}
  at ./lint.pl line 30, fr.po line 675.
Difference between msgid="Failed to delete remote registry {name}" and msgstr="Échec de la suppression du registre distant {nom}":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 961.
Difference between msgid="Failed to {operation} tag "{tag}"." and msgstr="Échec de {opération} du tag: "{tag}".":
  Missing from msgstr: {operation}
  at ./lint.pl line 30, fr.po line 969.
Difference between msgid="Indexing execution environments in {name}" and msgstr="Indexation des environnements d'exécution dans {nom}":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 1116.
Difference between msgid="Indexing failed for {name}" and msgstr="L'indexation a échoué pour {nom}":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 1124.
Difference between msgid="Provided by {company}" and msgstr="Fourni par {entreprise}":
  Missing from msgstr: {company}
  Unexpected in msgstr: {entreprise}
  at ./lint.pl line 30, fr.po line 1604.
Difference between msgid="Success: {name} was deleted" and msgstr="Succès : {nom} a été supprimé":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 2032.
Difference between msgid="Successfully deleted remote registry {name}" and msgstr="Suppression réussie du registre distant {nom}":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 2058.
Difference between msgid="Sync failed for {name}" and msgstr="Sync a échoué pour {nom}":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 2096.
Difference between msgid="Sync initiated for {name}" and msgstr="Sync initiée pour {nom}":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 2108.
Difference between msgid="Use the <0>Repository Management</0> page to sync collections curated by your organization to the Red Hat Certified repository in your private Automation Hub. Users with the correct permissions can use the sync toggles on the <1>Collections</1> page to control which collections are added to their organization's sync repository." and msgstr="Utilisez la page <0>Gestion de référentiel</0> pour synchroniser les collections créées par votre organisation avec le référentiel Red Hat Certified dans votre Automation Hub privé. Les utilisateurs disposant des autorisations appropriées peuvent utiliser les boutons de synchronisation de la page Collections pour contrôler les collections qui sont ajoutées au référentiel de synchronisation de leur organisation.":
  Missing from msgstr: <1> </1>
  at ./lint.pl line 30, fr.po line 2338.
Difference between msgid="{company} logo" and msgstr="logo {entreprise}":
  Missing from msgstr: {company}
  Unexpected in msgstr: {entreprise}
  at ./lint.pl line 30, fr.po line 2632.
Difference between msgid="{name} will be cancelled." and msgstr="{nom} sera annulé.":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 2640.
Difference between msgid="{name} will stop running." and msgstr="{nom} va cesser de fonctionner.":
  Missing from msgstr: {name}
  Unexpected in msgstr: {nom}
  at ./lint.pl line 30, fr.po line 2643.
```

**Comment by Spredzy** [comment]
Superseded by https://github.com/ansible/ansible-hub-ui/pull/1187

**Pull request closed without merging**